mmdet/models/utils/res_layer.py

```diff
@@ -0,0 +1,101 @@
+from torch import nn as nn
+
+from mmdet.models.utils import build_conv_layer, build_norm_layer
+
+
+class ResLayer(nn.Sequential):
+
+    def __init__(self,
+                 block,
+                 inplanes,
+                 planes,
+                 blocks,
+                 stride=1,
+                 dilation=1,
+                 style='pytorch',
+                 avg_down=False,
+                 with_cp=False,
+                 conv_cfg=None,
+                 norm_cfg=dict(type='BN'),
+                 dcn=None,
+                 gcb=None,
+                 gen_attention=None,
+                 gen_attention_blocks=[],
+                 **block_kwargs):
```

Comment:
I suggest passing all arguments other than the ones that are not used by the res block as **kwargs.